linode/manager
# feat: [UIE-8082] - DBaaS GA Database Create: Summary section, refactoring

## Description 📝
DBaaS GA: Database Create: Summary Section, refactoring

## Changes  🔄
List any change relevant to the reviewer.
- Added Summary section for Database Create
- Refactoring Database Create

## Target release date 🗓️
11/12/24

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-10-31 at 12 13 40 PM](https://github.com/user-attachments/assets/5b1334b9-7764-41f9-8bc0-04f425af7b61) | ![Screenshot 2024-10-31 at 12 14 14 PM](https://github.com/user-attachments/assets/72b939ec-6624-4595-b9d7-f442f6784fd2) |
| | ![Screenshot 2024-10-31 at 12 14 27 PM](https://github.com/user-attachments/assets/42ecbb0e-b54e-4403-9caa-04c9e949f111) |

## How to test 🧪

### Prerequisites
(How to setup test environment)
- Managed Databases account capability
- LD flag: beta: false, enabled: true

### Verification steps
(How to verify changes)

- Navigate to the Database Create page
- Summary section is displayed at the bottom of the page with the default text
- Select a Plan and the Number of Nodes.
- Summary section updates to display the selected plan, number of nodes, and price.

## As an Author I have considered 🤔

*Check all that apply*
